src/features/landing/components/ForWhoSection/ForWhoSection.tsx

```
@@ -1,27 +1,54 @@
 
+  const decorativeImages = [
+    { src: circle, sx: styles.circle },
+    { src: rhombus, sx: styles.rhombus },
+    { src: pointGroup, sx: styles.pointGroup },
+    { src: point6, sx: styles.point6 },
+    { src: point8, sx: styles.point8 },
+    { src: point10, sx: styles.point10 },
+    { src: waves, sx: styles.waves },
+    { src: hexagon, sx: styles.hexagon },
+    { src: triangle, sx: styles.triangle }
+  ];
```

ESLint

[error] Expected decorativeImages to have a type annotation.

(@typescript-eslint/typedef)